scripts/contributing.js

```diff
@@ -36,9 +38,11 @@
                 labels: 'beginner-friendly'
             },
             success: function(data){
-                for(var i = 0; i < data.length; i++){
-                    $('.open-issues-section--beginner .issues').append('<li class="issue"><a href="' + data[i].html_url + '"><span class="issue__number">#' + data[i].number +'</span><span class="issue__title">' + data[i].title + '</span></a></li>');
-                }
+                var limitedData = _.take(data, ISSUES_LIMIT);
+
+                _.forEach(limitedData, function(issue) {
```

Comment:
hm, i suppose none. at least, nothing intentional. i see no reason to use lodash over it -- i'll change it